docs/modules/user-guide/nav.adoc

```diff
@@ -1,32 +1,36 @@
 * xref:index.adoc[]
-* xref:devfile-registry.adoc[]
 
-* xref:authoring-stacks.adoc[]
-** xref:adding-schema-version-to-a-devfile.adoc[]
-** xref:adding-a-name-to-a-devfile.adoc[]
-** xref:adding-projects-to-a-devfile.adoc[]
-** xref:adding-commands-to-a-devfile.adoc[]
-** xref:adding-components-to-a-devfile.adoc[]
-*** xref:adding-kubernetes-component-to-a-devfile.adoc[]
-*** xref:adding-container-component-to-a-devfile.adoc[]
-*** xref:specifying-persistent-storage.adoc[]
-*** xref:limiting-resources-usage.adoc[]
-*** xref:defining-environment-variables.adoc[]
-*** xref:defining-endpoints.adoc[]
-*** xref:defining-kubernetes-resources.adoc[]
+* Developer
+** xref:using-stacks.adoc[]
+*** xref:adding-schema-version-to-a-devfile.adoc[]
+*** xref:adding-a-name-to-a-devfile.adoc[]
+** xref:authoring-stacks.adoc[]
+*** xref:adding-projects-to-a-devfile.adoc[]
+*** xref:adding-commands-to-a-devfile.adoc[]
+*** xref:adding-components-to-a-devfile.adoc[]
+**** xref:adding-kubernetes-component-to-a-devfile.adoc[]
+**** xref:adding-container-component-to-a-devfile.adoc[]
+**** xref:specifying-persistent-storage.adoc[]
+**** xref:limiting-resources-usage.adoc[]
+**** xref:defining-environment-variables.adoc[]
+**** xref:defining-endpoints.adoc[]
+**** xref:defining-kubernetes-resources.adoc[]
 
-** xref:adding-attributes-to-a-devfile.adoc[]
-** xref:adding-event-bindings.adoc[]
-** xref:referring-to-a-parent-devfile-in-a-devfile.adoc[]
+*** xref:adding-attributes-to-a-devfile.adoc[]
+*** xref:adding-event-bindings.adoc[]
+*** xref:referring-to-a-parent-devfile-in-a-devfile.adoc[]
 
-* xref:migrating-to-devfile-v2.adoc[]
-** xref:migrating-schema-version.adoc[]
-** xref:migrating-projects.adoc[]
-** xref:migrating-components.adoc[]
-** xref:migrating-plug-ins.adoc[]
-** xref:migrating-commands.adoc[]
-** xref:adding-event-bindings.adoc[]
-** xref:referring-to-a-parent-devfile-in-a-devfile.adoc[]
+** xref:migrating-to-devfile-v2.adoc[]
+*** xref:migrating-schema-version.adoc[]
+*** xref:migrating-projects.adoc[]
+*** xref:migrating-components.adoc[]
+*** xref:migrating-plug-ins.adoc[]
+*** xref:migrating-commands.adoc[]
+*** xref:adding-event-bindings.adoc[]
+*** xref:referring-to-a-parent-devfile-in-a-devfile.adoc[]
+
+* Registry Administrator
```

Comment:
We can add the rest of the devfile registry docs here once the TOC changes get merged.